app/src/test/java/mozilla/lockbox/presenter/AccountPresenterTests.kt

```diff
@@ -0,0 +1,57 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package mozilla.lockbox.presenter
+
+import android.content.Context
+import junit.framework.Assert.assertEquals
+import mozilla.lockbox.R.string.button_summary
+import mozilla.lockbox.R.string.button_title
+import mozilla.lockbox.R.string.disconnect_account_summary
+import mozilla.lockbox.R.string.email_example
+import mozilla.lockbox.adapter.AccountConfiguration
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.RuntimeEnvironment
+
+@RunWith(RobolectricTestRunner::class)
+class AccountPresenterTests {
+
+    private lateinit var context: Context
+    private val accountView = AccountViewFake()
+    private val subject = AccountPresenter(accountView)
+
+    class AccountViewFake : AccountView {
```

Comment:
Also: looking around, every other test uses `FakeX`; use that instead of `XFake`.